demo/addons/fmod/fmod.gdextension

```diff
@@ -1,6 +1,7 @@
 [configuration]
 entry_symbol = "fmod_library_init"
 compatibility_minimum = 4.2
```

Comment:
This also means when upgrading to Godot 4.5 we need to bump 4.5 because it seems 4.4 bindings are not compatible with 4.5